test/cluster/test_tablets_merge.py

```
@@ -517,14 +533,14 @@ async def test_merge_with_drop(manager: ManagerClient, feature_config: FeatureCo
             }
-    cfg = feature_config.get_cluster_cfg(cfg)
+    cfg = storage_config.get_cluster_cfg(feature_config.get_cluster_cfg(cfg))
     cmdline = [
         '--logger-log-level', 'load_balancer=debug',
         '--logger-log-level', 'debug_error_injection=debug',
     ]
     server = await manager.server_add(cmdline=cmdline, config=cfg)
-    keyspace_opts = feature_config.get_keyspace_opts(
-        f"WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}}")
+    keyspace_opts = storage_config.get_keyspace_opts(feature_config.get_keyspace_opts(
+        f"WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}}"))
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)